models/classes/search/SearchProxy.php

```diff
@@ -41,6 +41,7 @@ class SearchProxy extends ConfigurableService implements Search
     public const OPTION_ADVANCED_SEARCH_CLASS = 'advanced_search_class';
     public const OPTION_DEFAULT_SEARCH_CLASS = 'default_search_class';
     public const OPTION_GENERIS_SEARCH_WHITELIST = 'generis_search_whitelist';
+    public const SAFE_NODES = ['http://www.tao.lu/Ontologies/generis.rdf#ClassRole'];
```

Comment:
You can import GenerisRdf:
```php
use oat\generis\model\GenerisRdf;
```
and use constant value instead
```suggestion
    public const SAFE_NODES = [GenerisRdf::CLASS_ROLE];
```

Reply:
fixed, thanks! :partying_face: